src/main/java/sirius/biz/web/BlobPdfReplaceHandler.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.web;
+
+import org.xhtmlrenderer.extend.FSImage;
+import org.xhtmlrenderer.extend.UserAgentCallback;
+import sirius.biz.storage.layer1.FileHandle;
+import sirius.biz.storage.layer2.BlobStorage;
+import sirius.kernel.commons.Strings;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.Register;
+import sirius.web.templates.pdf.handlers.PdfReplaceHandler;
+
+import javax.annotation.Nullable;
+import java.util.Optional;
+
+/**
+ * Resolves blob:// URIs to resized images while maintaining the image ratios.
+ * <p>
+ * The format of the URI needs to match blob://space/variant/blobKey.
```

Comment:
shouldn't this be blob://space/blobKey/variant?

Also do we plan to support readable URLs like /pub/my_magic_image.jpg?

Reply:
This is how line 49 implements it. I can switch it around if you'd like.

Readable URLs would currently only make sense for the public space, which is not part of biz.